src/items.rs

```diff
@@ -1118,6 +1118,18 @@ pub(crate) fn format_trait(
     }
 }
 
+struct GenericBounds<'a> {
```

Comment:
Nit: now that this type is specific to opaque types, I prefer to rename it to something like `OpaqueTypeBounds`.

Reply:
That's certainly true. I renamed it.